sdk/storage/storage-blob-changefeed/src/AvroReaderFactory.ts

```diff
@@ -0,0 +1,23 @@
+import { AvroReadable, AvroReader } from '../../storage-internal-avro/src';
+
+export class AvroReaderFactory {
+  public buildAvroReader(dataStream: AvroReadable): AvroReader;
```

Comment:
minor: why it's not simply called 'create'? Personal perference although.

Reply:
following .Net. Not sure about the reasoning though.